tests/unit/deploy/test_deployer.py

```diff
@@ -607,3 +609,284 @@ def badview():
 
     with pytest.raises(ValueError):
         validate_routes(sample_app.routes)
+
+
+class TestLambdaInitialDeploymentWithConfigurations(object):
+    @fixture(autouse=True)
```

Comment:
Ah then please call it something else.  I think people seeing a setup() function in a test class would expect it to behave like unittest.TestCase and wonder why that's not being used instead.  Might be helpful to add a comment as well.